src/sentry/options/defaults.py

```diff
@@ -287,6 +287,22 @@
     flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Killswitch to enable id or slug path parameters for some endpoints
+register(
+    "api.id-or-slug-enabled-ea",
+    default=False,
+    type=Bool,
+    flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE,
+)
+
+# Killswitch to enable id or slug path parameters for some endpoints for certain organizations, if available
+register(
+    "api.id-or-slug-enabled-ea-org",
+    type=Sequence,
+    default=[],
+    flags=FLAG_ALLOW_EMPTY | FLAG_AUTOMATOR_MODIFIABLE,
+)
+
```

Comment:
I'm not familiar with the flags here 😶‍🌫️  @kneeyo1 can you please double check?

Reply:
I believe this is correct, I used this for my staff option

Reply:
these make sense to me!